test/integration/use_int_test.go

```diff
@@ -121,13 +122,28 @@ func (suite *UseIntegrationTestSuite) TestReset() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
-	cp := ts.SpawnWithOpts(e2e.WithArgs("checkout", "ActiveState-CLI/Python3"))
+	cp := ts.SpawnWithOpts(
+		e2e.WithArgs("checkout", "ActiveState-CLI/Python3"),
+		e2e.AppendEnv("SHELL=bash"),
+	)
 	cp.Expect("Skipping runtime setup")
 	cp.Expect("Checked out project")
 	cp.ExpectExitCode(0)
 
+	// Create a zsh RC file to ensure that the path is updated for both bash and zsh.
+	var zshRcFile string
+	var err error
+	if runtime.GOOS != "windows" {
+		zsh := &zsh.SubShell{}
```

Comment:
This test case (TestReset) is very over-loaded. Let's not make this problem worse and instead address the shell tests with their own test rather than piggy backing off of this one.